### PR TITLE
Make the title of the admin interface translatable

### PR DESCRIPTION
**Pull request by PeterW-LWL**

### CODE REVIEW

File: filer/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -45,6 +45,11 @@ msgid_plural "All %(total_count)s selected"
 msgstr[0] "%(total_count)s ausgewählt"
 msgstr[1] "Alle %(total_count)s ausgewählt"
 
+#: admin/folderadmin.py:431
```
Comment (yakky) [inline filer/locale/de/LC_MESSAGES/django.po]: Could you remove this? Translations are handled on transifex and we cannot merge translations from github

**Comment by PeterW-LWL** [comment]
I addressed the the comment. Should I squash the commits?

**Comment by yakky** [comment]
@czpython @stefanfoulis could you have a look at the mac builds failures?

**Comment by PeterW-LWL** [comment]
@yakky @stefanfoulis travis seems to have some bigger problems with their mac builds...

**Comment by stefanfoulis** [comment]
I'm fine with removing setting the Mac builds as "allowed failures" in the matrix build for now. Seems to be a problem on travis.

**Comment by yakky** [comment]
@PeterW-LWL sorry it took so much time managing this. Could you please rebase this on top of current develop as tests are now working correctly

**Comment by PeterW-LWL** [comment]
Rebased.

**Comment by yakky** [comment]
@PeterW-LWL thanks. Hopefully tests will pass now and we're good to merge

**Comment by yakky** [comment]
Yay, we made it! Thanks @PeterW-LWL for your contribution

**Pull request merged**